README.md

```diff
@@ -50,7 +50,7 @@ To discuss the projects and the content under the Plutonomicon umbrella, join ou
 
 If you'd like to run a live version of the website locally:
 
-```bash
+```sh-session
```

Comment:
What is all this?

Reply:
That's language identifier for shell-session. cf. https://github.com/PrismJS/prism/pull/1892